package.json

```diff
@@ -4,7 +4,7 @@
   "description": "React components for Wonder Blocks design system.",
   "main": "index.js",
   "scripts": {
-    "install": "for d in packages/*/ ; do (cd \"$d\" && npm install); done",
+    "postinstall": "for d in packages/*/ ; do (cd \"$d\" && npm install); done",
```

Comment:
I changed this to `postinstall` since the `prepare` scripts in each of the sub packages depends on some of the build tools that are installed at the base level.